src/z3c/form/converter.rst

```diff
@@ -1083,6 +1083,42 @@ If the list is empty, the missing value is returned:
   >>> conv.toFieldValue([]) is None
   True
 
+Turns out, we can have fields that accept more types.
+
+  >>> @zope.interface.implementer(zope.schema.interfaces.IList)
+  ... class MySequence(zope.schema._field.AbstractCollection):
+  ...    _type = (list, tuple)
+
+  >>> numbers = MySequence(
+  ...     value_type=zope.schema.Int(),
+  ...     default=[],
+  ...     missing_value=None,
+  ...     )
+
+  >>> from z3c.form.browser import multi
+  >>> multiWidget = multi.MultiWidget(TestRequest())
+  >>> multiWidget.field = numbers
+
+We now use the field and widget to instantiate the converter:
+
+  >>> conv = converter.MultiConverter(numbers, multiWidget)
+
+We can now convert a list or tuple of integers to the multi-widget internal
+representation:
+
+  >>> conv.toWidgetValue([1, 2, 3, 4])
+  [u'1', u'2', u'3', u'4']
+
+  >>> conv.toWidgetValue((1, 2, 3, 4))
+  [u'1', u'2', u'3', u'4']
+
+Now, let's look at the reverse. We get a tuple because that's the last
+type in `_type`
+
+  >>> conv.toFieldValue([u'1', u'2', u'3', u'4'])
+  (1, 2, 3, 4)
```

Comment:
Why are you asserting the list is getting converted to a tuple?

Reply:
Oh, didn't notice the comment above. Anyways, why last type in _type is special? Is it a feature or just conincidence?

Reply:
the list is the input, coming from the request, the return value is the value we want to use in our app -- the type is the last from _type, tuple in this case

Reply:
why the last type in _type is special? Is it a feature or just a conincidence?

Reply:
"copy what the others do" `TextLinesConverter` and `CollectionSequenceDataConverter` does the same, I'd stick to that convention (fixed the .rst's a bit)